### DISCUSSION TITLE
Nested Article Urls

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7393] [General]
I'm getting really confused. I've watched the Laracast Videos etc on creating blog posts, however he only does them 1 level deep. So i'm confused whether to create Collections or pages and how to bind them to create nested posts. 

How do you categorize them with a similar URL below. Is it with taxonomies or with orderable nested collections via drag an drop?

I'm trying to create blog articles with SEO urls like /resources/{main_category}/{the_sub_category}/{the_article}.

However when he creates a collection then binds them to a page, then adds the taxonomy when you view the show page, the page is only bound to the root url "resources" and not the taxonomy.

I want urls like /resources/{main_category}/{the_sub_category}/{the_article} but don't know how to achieve this.

**Comment by jackmcdade** [comment]
There is no native way to do this at the moment. We’d love to solve it in the future though!On Jan 22, 2023, at 9:57 PM, ottz0 ***@***.***> wrote:﻿
I'm getting really confused. I've watched the Laracast Videos etc on creating blog posts, however he only does them 1 level deep. So i'm confused whether to create Collections or pages and how to bind them to create nested posts.
How do you categorize them with a similar URL below. Is it with taxonomies or with orderable nested collections via drag an drop?
I'm trying to create blog articles with SEO urls like /resources/{main_category}/{the_sub_category}/{the_article}.
However when he creates a collection then binds them to a page, then adds the taxonomy when you view the show page, the page is only bound to the root url "resources" and not the taxonomy.
I want urls like /resources/{main_category}/{the_sub_category}/{the_article} but don't know how to achieve this.

**Reply by ottz0** [reply]
Is this something you could fast track? We were just looking to burn away our WordPress website and move our articles into Statamic as we have nested our articles inside of categories in wordpress SEO friendly as per above and now they will not be grouped (only via taxonomies). Eg: /transport/{some_type_of_transport} instead of /transport/cars/{taxis}.

Now I don't want to move away from Wordpress for this feature, as I don't want to create a new bunch of articles in statamic only to lose SEO and have to do a bunch of 301's when support may or many not happen in the future.

**Reply by jasonvarga** [reply]
If your entries are only assigned to a single category, you could set the collection's route to `/transport/{type}/{slug}` (where `type` is the name of your taxonomy) 

Since, if your taxonomy field is set to only save a single item, it'll just be the term's slug, so it can be included in the url.

If your entries can be assigned to multiple types, then it won't work because then you'd end up with an entry at multiple urls, which isn't supported natively at the moment.

**Comment by ottz0** [comment]
I created a new collection (articles) and set the collection to orderable with the URL route pattern set to `/{parent_uri}/{slug}`. This will then give me the SEO URL's that I need to categorize my articles. I then created 2 blueprints within the collection (Article Section, Article Single) and set both of those to each on my Index and show pages with will then bind my blueprints to the views.

I can then use my taxonomies not to order my structure but to use them like "mini tags" like for keywords etc. This is similar to wordpress.
